plasma_framework/contracts/src/exits/payment/PaymentTransactionStateTransitionVerifier.sol

```diff
@@ -8,10 +8,14 @@ import "../../transactions/WireTransaction.sol";
 import "../../transactions/PaymentTransactionModel.sol";
 import "../../transactions/outputs/PaymentOutputModel.sol";
 
```

Comment:
https://github.com/omisego/plasma-contracts/blob/406bed9da1316f8d75da31d458e0306467a3911c/plasma_framework/contracts/src/exits/payment/PaymentTransactionStateTransitionVerifier.sol#L32 could we remove the todo and open an issue for this on our gh if it does not exist already?

Reply:
callout~ @pgebal can you help me understand what's the original issue on this? I think I used to understand what it means but not now ><

Reply:
@boolafish  @thec00n the issue is that bytes array from calldata can not be copied to memory. So I had to consume inputTxs in the function body, couldn't pass it as argument.